### ISSUE TITLE
Release of 1.0

### ISSUE THREAD

**Issue by matze** [ufo-kit/ufo-core#165]
This is a meta issue discussing things necessary for the release of version 1.0 that does not allow any API removal or backwards-incompatible change. Of course all tickets marked with a milestone 1.0 should be fixed but there are a few things worth discussing:

- [ ] Should we drop CMake with 1.0 or later?
- [ ] Which issues really block release of 1.0, i.e. which require API changes and …
- [ ] … which issues could we postpone because they do not affect API stability?

**Comment by tfarago** [comment]
Oh wow! This is big. Regarding the fist point, if we keep CMake does it mean that we have to keep it forever? It's not an API-related thing, so I don't know how strict the rules are here.

**Comment by matze** [comment]
> Regarding the fist point, if we keep CMake does it mean that we have to keep it forever? It's not an API-related thing, so I don't know how strict the rules are here.

No, that's just infrastructure. We can drop it any time but it would be good time instead of keeping it around forever. Although we should really check if someone is *not* able to use meson/ninja.

**Comment by tfarago** [comment]
@milkdrop and @lucito, have you tried that build system already? If not, can you please?

**Comment by matze** [comment]
Wait a second, what means 1.0 for you? Tomas was just asking if compiling via `meson build` instead of `cmake ..` would make a difference to you?

**Comment by milkdrop** [comment]
Ahhh, I see.. Well, I have removed old ufo-core and installed this release using cmake. And it does not work. So I rolled back to April release and it works as it should again. Do I have to reinstall all dependents each time I update core? 

I had no clue about existence of that meson system, but I will give it a try of course.

Cheers,
Sergei.

**Comment by matze** [comment]
> Please tell me output of which command would you need for troubleshooting.

That line should be enough, can you replicate the issue @tfarago?

**Comment by tfarago** [comment]
Hi, which versions of core and filters do you use @milkdrop exactly? Please type `git log` in the ufo-core and ufo-filters directories and let us know what the last commit is.
As for me, I am happily flatcorrecting away with 9ac9a036fd08298afb29084a8c22b495385add2d

**Comment by milkdrop** [comment]
Hola, let me try to update all modules to the latest versions, I'm pretty sure the problem will disappear.  BTW about that _meson_ system - as I've said it is totally new to me, it would be great to give at least basic directives in the appropriate section of the documentation http://ufo-core.readthedocs.io/en/latest/install/linux.html#installing-from-source
